index.bs

```diff
@@ -15,19 +15,107 @@ Complain About: accidental-2119 yes, missing-example-ids yes
 Markup Shorthands: markdown yes, css no
 Assume Explicit For: yes
 </pre>
+<pre class="anchors">
+spec: HTTP; urlPrefix: https://httpwg.org/specs/rfc7231.html#
+    type: dfn; text: HTTP 3xx statuses; url: status.3xx
+spec: tracking-dnt; urlPrefix: https://www.w3.org/TR/tracking-dnt/#
+    type: dfn; text: tracking; url: dfn-tracking
+</pre>
 
 <section class="non-normative">
 <h2 id="intro">Introduction</h2>
 
 <em>This section is non-normative.</em>
 
-Blah blah blah.
+Browsers are working to prevent cross-site [=tracking=], which threatens user
+privacy. In addition to third-party cookies and storage, other client-side
+methods exist that enable cross-site tracking. [=Navigational tracking=]
+correlates user identities across sites during navigations between those sites.
+[=Navigational tracking=] uses [=link decoration=] to convey information, but
+not all [=link decoration=] is tracking. This project attempts to distinguish
+tracking from non-tracking navigation and to prevent the tracking without
+damaging similar but benign navigations.
+
 </section>
 
 <h2 id="infra">Infrastructure</h2>
 
 This specification depends on the Infra standard. [[!INFRA]]
 
+# Terminology # {#terminology}
+
+<dfn>Link decoration</dfn> is when the source of a [=hyperlink=] "decorates" its [=URL=]
+with extra information beyond what's necessary to identify the page a user wants
+to navigate to. This information can be placed almost anywhere inside the URL.
+
+<dfn>Navigational tracking</dfn> refers to the general use of a
+[[HTML#browsing-the-web|navigation]] to identify that a user on one site is the
+same person as a user on another site. Navigations are usually visible to users,
+but it's possible for a site to navigate away and redirect back fast enough that
+only attentive users would notice.
+
+<div class="example" id="example-link-decoration-tracking">
+<style>
+#example-link-decoration-tracking code em {background-color: cyan}
+</style>
+
+Examples and non-examples of [=link decoration=] and [=navigational tracking=],
+with the potential decoration or tracking element emphasized:
+
+: <code>https://publisher.example/page?userId=<em>5789rhkdsaf8urfnsd</em></code>
+
+:: [=Link decoration=], and also [=navigational tracking=].
+
+: <code>https://bookshop.org/a/<em>1122</em>/9780062252074</code>
+
+:: [=Link decoration=] but **not** [=navigational tracking=]: This number
+    identifies an affiliate to credit with a book sale. Replacing this with
+    another number gets to the same target page.
+
+: <code>https://bookshop.org/a/1122/<em>9780062252074</em></code>
+
+:: **Not** decoration: This number identifies a particular book. Changing it
+    yields a different target page.
+
+: <code>https://bugzilla.mozilla.org/show_bug.cgi?id=<em>1460058</em></code>
+
+:: **Not** decoration: changing the number changes which bug the user sees.
+
+: <code>https://www.google.com/maps/@<em>37.4220328,-122.0847584,17.12z</em></code>
```

Comment:
I suggest removing this example, or at least describing it differently, as it conflates two things.   Changing the numbers could change the content of the destination page _and also uniquely describe the user_.

Reply:
I think it's an important example for demonstrating what's hard about navigational-tracking mitigations, but I'm happy to describe it differently.

In this case, the numbers do not encode user-identifying information, and modifying them to embed a user ID wouldn't successfully communicate a user ID to anyone (since nobody's listening within google.com/maps to decode the user ID). But it's hard for an automated system inside a browser to prove that, and even hard for humans reading the URL to be confident of it.

Is that the better description? That there's enough information here (10+ digits) to identify a user, but it's used to identify a place instead, and removing enough digits to prevent it from identifying a user would prevent it from serving its purpose of identifying the right place?

Reply:
What would you think of calling it an "ambiguous" case, with additional text similar to what you've written above?